lib/model.js

```diff
@@ -758,6 +758,14 @@ module.exports = function(registry) {
     var pathName =
       (scope.options && scope.options.http && scope.options.http.path) || scopeName;
 
+    var updateOnlyProps = this.getUpdateOnlyProperties();
+    var createOnly = false;
```

Comment:
Let's use a more descriptive name please, e.g. `hasUpdateOnlyProps`.

Also please use `const` instead of `var`, see http://loopback.io/doc/en/contrib/style-guide.html#variable-declarations

```js
const updateOnlyProps = this.getUpdateOnlyProperties();
const hasUpdateOnlyProps = updateOnlyProps && updateOnlyProps.length > 0;
```